_includes/header.html

```diff
@@ -55,8 +55,10 @@
             <li role="separator" class="divider"></li>
             <li><a href="/elements-code-tutorial/easy-run-code">An easy way to run the main tutorial code</a></li>
             <li><a href="/elements-code-tutorial/advanced-examples">Advanced examples</a></li>
+            <li><a href="/elements-code-tutorial/building-elements-from-source">Building Elements from source</a></li>	    
             <li role="separator" class="divider"></li>
-            <li><a href="https://github.com/ElementsProject/elements">Elements on Github</a></li>
```

Comment:
If you remove this then the last line in the menu is a line separator - so remove the separator above as well looks neater or else you get a double end to the menu.

Reply:
did not mean to delete that, will restore.